core/field_angle.ts

```diff
@@ -198,7 +198,7 @@ export class FieldAngle extends FieldTextInput {
     // #2380)
     this.symbol_ = dom.createSvgElement(Svg.TSPAN, {});
     this.symbol_.appendChild(document.createTextNode('\u00B0'));
-    this.textElement_.appendChild(this.symbol_);
+    this.textElement_!.appendChild(this.symbol_);
```

Comment:
Discussed w/ Maribeth, and using `!.` in cases like this is fine b/c we know it was created in `initView`